en/log/plotjuggler_log_analysis.md

```diff
@@ -0,0 +1,101 @@
+
+This is demonstrated in an animation below:
+
+![Plot Juggler Multi panel demonstration](../../assets/flight_log_analysis/plot_juggler/plotjuggler_dragdrop_multipanel.gif)
+
+In this example, `vehicle_local_position` topic's `ax`, `ay` and `az` (Acceleration estimate) components were graphed first by splitting the screen in 3 sections.
+Then, the `vz` (Velocity estimate) component was added under right pane, and finally `battery_status` topic's `current_a` (Battery current) was graphed in the lower-mid pane.
+
+Although not obvious in the beginning you can observe that whenever the vehicle starts moving (battery current value goes high), the acceleration and velocity values start to move around as well.
+This is because all the data is displayed as a time-series, which shows each value at on a specific timestamp.
+
+This is useful for providing a broad view of what happened and why.
+It is often hard to troubleshoot a problem just by looking at one graph, but by having multiple graphs displayed, it is much easier to figure out what was going on in the system.
+
+### Displaying Data in 2D
+
+Another powerful feature that exists is the ability to display 2D data on a XY plane (each data on X, Y axis) in a scatter-plot manner.
+This is done by selecting two data points by holding `Ctrl` key while selecting each of them (e.g. `vehicle_local_position` topic's `x` and `y` components), and drag & dropping it with the `Right mouse cursor` pressed.
+
+![Plot Juggler 2D graphing](../../assets/flight_log_analysis/plot_juggler/plotjuggler_2d_graph.gif)
+
+In this example, the estimated vehicle's position in local coordinate frame is graphed onto a XY-plane which shows a 2D view of the estimated position, and the `vx` and `vy` components (Velocity estimate) are graphed on the right, with the `vz` (Vertical velocity estimate) graphed underneath it in a split view.
+
+This in turn shows the relationship between position and the velocity of the vehicle intuitively.
+For example, note how when the vehicle moves in the X-axis direction, the `vx` value goes high, and as the vehicle turns to the Y-axis direction, the `vy` value starts to change as well.
+
+#### Using the 'Play' button
+
+Here the **Play** button is used to play the recorded data in real time (speed factor can be adjusted in the lower-right corner).
+This shows the position / velocity relationship described above in detail.
+
+![Plot Juggler 2D in-depth analysis](../../assets/flight_log_analysis/plot_juggler/plotjuggler_2d_graph_pos_vel_analysis.gif)
+
+### Layout Templates
+
+There are a number of PlotJuggler layout files shared by PX4 Developers.
+Each can be used for a specific purpose (Multicopter tuning, VTOL tuning, Boat debugging, etc.):
+
+* [Sample View layout](/assets/flight_log_analysis/plot_juggler/plotjuggler_sample_view.xml) : Template used in the [Multi-panel example](#splitting-horizontally--vertically--multi-panel) above
+* [VTOL Tuning layout]() 
+* [Multicopter Tuning layout]() 
```

Comment:
missing links

Reply:
I asked Matthais and Silvan for this!

Reply:
@MaEtUgR @sfuhrer Can you ETA help for this?

IF not, perhaps we comment these out while waiting?

Reply:
Turns out, @sfuhrer is the only one with the layouts, @MaEtUgR creates the layout each time, so we can delay this part for now.

Reply:
Correct, I have no templates 😇 Silvan will be back on Thursday 😃